Allow specifying preprocessor order (#1607)
rust-lang/mdBook: guide/src/format/configuration/preprocessors.md

[preprocessor.random]
command = "python random.py"
```

### Require A Certain Order

> The order in which preprocessors are run is not guaranteed, but you can request some to run before or after others.

Reviewer: I think this will be guaranteeing a certain order, since they will be sorted.  Perhaps this introduction could be changed to something like this?

```suggestion
The order in which preprocessors are run can be controlled with the `before` and `after` fields.
```
